gen/templates/datasource_test.go.tmpl

```diff
@@ -4,31 +4,70 @@
 
 package provider
 
+{{- $found_cert := false -}}
+{{- range $key, $value := $.all}}
+  {{- if contains $key "cert" }}
+    {{- $found_cert = true -}}
+  {{- end}}
+{{- end}}
+
 import (
 	"testing"
 	"regexp"
+	"time"
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	{{- if $found_cert }}
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	{{- end}}
 )
 
 {{- if .parents}}
 	{{- range .parents}}
 func TestAccDataSource{{$.resourceClassName}}With{{capitalize .class_name}} (t *testing.T) {
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { testAccPreCheck(t) },
+		PreCheck:                 func() { 
+		time.Sleep(10 * time.Second)
```

Comment:
Is this for the reload issue on .70? Should we do this sleep only for the resource/datasources needed, else there would be 10 second per test added.

Reply:
Yeah, this was to avoid the test running after this one to not face the same reload issue, but have come up with a different procedure now